openmetadata-spec/src/main/resources/json/schema/entity/data/topic.json

```diff
@@ -51,6 +51,84 @@
         }
       },
       "additionalProperties": false
+    },
+    "dataTypeTopic": {
+      "javaType": "org.openmetadata.schema.type.FieldDataType",
```

Comment:
@harsha, do you want to create a separate type called schema? We can reuse this for Avro, JSON, Protobuf, etc., right?

Would this be useful for Mongodb and elasticsearch as well?